crates/sui-indexer/migrations/2023-08-19-044044_checkpoints/up.sql

```diff
@@ -25,4 +25,5 @@ CREATE TABLE checkpoints
 );
 
 CREATE INDEX checkpoints_epoch ON checkpoints (epoch);
+CREATE INDEX checkpoints_epoch_sequence_number ON checkpoints (epoch, sequence_number);
```

Comment:
can you change `checkpoints_epoch` instead of making a new index, so that both queries of `where epoch = a` and `where epochs = a and cp = b` can leverage that.

Reply:
ah yeah that's fair, since we'll always be ordering by `sequence_number`, so we aren't going to have a query that is purely `select * from checkpoints where epoch=100 limit 50;`?

Reply:
updated